### PR TITLE
docs: update local dev server README

### PR DESCRIPTION
**Pull request by cgund98**
## **Description**

- Updated `apps/web/README.md` to have up-to-date description of the monolith app.  See the new README [here](https://github.com/waldo-vision/waldo/tree/fix/update-web-docs/apps/web). Also updated the getting started instructions in the docs website.
- Made a couple of tweaks to `docker-compose.yml` to fix a couple of issues I ran into while starting the containers with SELinux enabled. 

## **Issues**

- N/A

---

- [x] I have read the [Contributing Guide](https://docs.waldo.vision/en/getting-started/), and agree to follow the [Code of Conduct](https://docs.waldo.vision/legal/code-of-conduct/).

### CODE REVIEW

File: apps/web/README.md [hunk 1]
```diff
@@ -1,34 +1,122 @@
...
+```bash
+# Build the app
+yarn workspace web build
+
+# Serve the app
+yarn workspace web start
+```
 
-Check out our [Next.js deployment documentation](https://nextjs.org/docs/deployment) for more details.
+You should now be able to visit the website at http://localhost:3000.
```
Comment (Huskydog9988) [inline apps/web/README.md:122]: id prefer all the docs go onto the docs site if possible. Im fine with quick reference stuff and a more specific breakdown of how this individual workspace works more deeply, but the setup stuff really should go on the getting started page.

File: docker-compose.yml [hunk 2]
```diff
@@ -10,7 +10,9 @@ services:
       # ui
       - 8081:8080
     volumes:
-      - './docker/cockroach/data:/cockroach/cockroach-data'
+      - './docker/cockroach/data:/cockroach/cockroach-data:z'
```
Comment (Huskydog9988) [inline docker-compose.yml:13]: why not make it `rw`?

File: apps/web/README.md (hunk 1 above)
Comment (Huskydog9988) [inline apps/web/README.md:122]: id prefer all the docs go onto the docs site if possible. Im fine with quick reference stuff and a more specific breakdown of how this individual workspace works more deeply, but the setup stuff really should go on the getting started page.

File: docker-compose.yml (hunk 2 above)
Comment (Huskydog9988) [inline docker-compose.yml:13]: why not make it `rw`?

**Review by Huskydog9988 — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 2 above)
Comment (cgund98) [reply docker-compose.yml:13]: Nope, just using vanilla docker. I've tried messing with the directory permissions but haven't had any luck. You can read more about the issue [here](https://projectatomic.io/blog/2015/06/using-volumes-with-docker-can-cause-problems-with-selinux/). Is there a significant concern with using this flag?

Comment (cgund98) [reply docker-compose.yml:13]: There isn't really a specific reason I used one over the other. I can switch to `Z` instead if we want to lock it down a bit more.

File: docs/src/pages/en/getting-started.md [hunk 3]
```diff
@@ -16,88 +16,169 @@ Before you can start hacking away on waldo, you need to have the tools below to
...
+
+```bash
+# start the container
+docker-compose up -d
+
+# Check that the container is running
+docker ps
+
+# If at any point you wish to shut down the container:
+docker-compose down
```
Comment (Huskydog9988) [inline docs/src/pages/en/getting-started.md:117]: can you use the compose v2 syntax here? compose v1 is currently deprecated.

File: docker-compose.yml (hunk 2 above)
Comment (Huskydog9988) [reply docker-compose.yml]: ok then its fine

**Review by Huskydog9988 — APPROVED** [review]

**Pull request merged**